homeassistant/components/zwave_js/helpers.py

```diff
@@ -399,6 +397,7 @@ def async_is_device_config_entry_not_loaded(
         raise ValueError(f"Device {device_id} not found")
     return any(
         (entry := hass.config_entries.async_get_entry(entry_id))
+        and entry.domain == DOMAIN
```

Comment:
Side note: The existing check here is incorrect. It doesn't take into account missing config entries.

```
Python 3.9.12 (main, Apr 18 2022, 22:40:46) 
Type 'copyright', 'credits' or 'license' for more information
IPython 7.29.0 -- An enhanced Interactive Python. Type '?' for help.

In [1]: any([None, None])
Out[1]: False
```

If no config entries are there for the device it will return False, which should mean that the config entries are loaded.

Reply:
good catch, thanks!